ale_linters/scala/metals.vim

```diff
@@ -1,24 +1,30 @@
 " Author: Jeffrey Lau - https://github.com/zoonfafer
 " Description: Metals Language Server for Scala https://scalameta.org/metals/
 
-call ale#Set('scala_metals_executable', 'metals-vim')
+call ale#Set('scala_metals_executable', 'metals')
```

Comment:
No idea how this works with Ale, but something important to keep in mind. Many Scala workspaces may have nested build files. Like multiple layers of `pom`s or even `build.sbt`. So it's a good idea to have logic to see if the pattern is found, does the parent also have that patter. If so, the parent should be the root, not the current one where the first layer of build file is found. You can see an an example of how I do this in [here](https://github.com/scalameta/nvim-metals/blob/main/lua/metals/rootdir.lua)